conans/test/functional/toolchains/cmake/test_cmake.py

```diff
@@ -730,3 +730,13 @@ def build(self):
 
         client.run("build . --profile=profile_false")
         assert "using FindComandante.cmake" in client.out
+
+
+@pytest.mark.tool("cmake")
+def test_cmake_verbosity():
+    tc = TestClient()
+    tc.run("new cmake_exe -d name=pkg -d version=1.0")
+    tc.run("create . -c tools.build:verbosity=verbose -c tools.compilation:verbosity=verbose")
+    assert "--loglevel=VERBOSE" in tc.out
+    assert "unrecognized option" not in tc.out
```

Comment:
Maybe a bit heavy test to just check the ``--verbose`` output?. Maybe it could be reduced, removing the actual compilation, and avoiding slow compiler checks instead of the ``cmake_exe`` template?

Reply:
Or integrate the checks in an existing test

Reply:
Yes, that's what I'm going to do, this test was done before the others, didn't rememebr to do the same as in them :)